src/test/java/ClientResourceTest.java

```diff
@@ -0,0 +1,131 @@
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.inject.Inject;
+import filemanager.configuration.ApplicationConfiguration;
+import filemanager.interceptor.RequestKeyCheckInterceptor;
+import filemanager.launcher.Launcher;
+import io.dropwizard.client.JerseyClientBuilder;
+import io.dropwizard.testing.junit.DropwizardAppRule;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Response;
+
+import static org.junit.Assert.assertEquals;
+
+public class ClientResourceTest {
+
+    @Inject
+    RequestKeyCheckInterceptor requestKeyCheckInterceptor;
+
+    @ClassRule
+    public static final DropwizardAppRule<ApplicationConfiguration> RULE = new DropwizardAppRule<>(Launcher.class,
+            "config.yml");
+
+    @After
+    public void tearDown() {
+        requestKeyCheckInterceptor = null;
+    }
+
+    @Before
+    public void setUp() {
+        requestKeyCheckInterceptor = new RequestKeyCheckInterceptor();
+    }
+
+    @Test
+    public void requestInterceptorKeyPassTest() {
+        Client client = new JerseyClientBuilder(RULE.getEnvironment()).build("test correct");
+
+        Response response = client.target(String.format("http://localhost:%d/main/check?keyPass=someValue", RULE.getLocalPort()))
```

Comment:
I think you may have something like ```baseUrl``` built once for the whole test, and then just add key with ```queryParam``` method on ```Target``` object.